source/about/cloud-subscriptions.rst

```diff
@@ -129,11 +129,11 @@ How do I cancel my subscription?
 To cancel your subscription, please `contact us <https://customers.mattermost.com/cloud/contact-us>`__.
 
 I'm no longer being billed for my Cloud Starter ($149 / year) plan. Why is that?
```

Comment:
@jasonblais / @johndavidlugtu: I wasn't sure how the naming changes/doesn't change for these customers.